docs/Third-party.md

```diff
@@ -1,281 +1,268 @@
 
-If you simply run the `export` and `source` commands in your terminal, these environment variables will be defined only for the duration of your session, and will be lost if you re-launch your terminal.
+<div class="grid">
+  <a href="Converters/ADNI2BIDS.md" class="card">adni-to-bids</a>
+  <a href="Converters/AIBL2BIDS.md" class="card">aibl-to-bids</a>
+  <a href="Converters/NIFD2BIDS.md" class="card">nifd-to-bids</a>
+  <a href="Converters/UKBtoBIDS.md" class="card">ukb-to-bids</a>
+</div>
 
-In order to define these variables permanently, you need to manually edit the configuration file associated to your shell.
+!!! warning "Version required"
+    Clinica requires dcm2niix version `1.0.20190902` or later.
 
-If you are using `bash`, it can be `~/.bashrc` or `~/.bash_profile`, if you are using `zsh`, it should be `~/.zshrc`.
+Please check the installation instructions for all platforms on [dcm2niix Git repository](https://github.com/rordenlab/dcm2niix#install).
 
-!!! note "test before"
-    Please do not copy/paste the provided commands without adapting them to your system and without testing them.
-    Most paths provided here require to be adapted to your system, and also depends on how you installed the software.
-    A good approach is to verify that the different paths you want to assign to a variable exist first.
-    If they do, then try running the `export` and `source` commands in your terminal, and verify that the software run as expected.
-    You can also verify that the pipeline you want to use is also running as expected.
-    If this works, then consider modifying your shell configuration file to have these variables automatically defined on every session.
 
 
-### ANTs
+## Pipeline-specific interfaces
 
-#### Installation
+<div class="annotate" markdown>
+Some, but not all pipelines use specific third-party software. Depending on your usage of Clinica, you will need to install additional packages.
+Specific dependencies are listed below (1) :
+</div>
 
-To install `ANTs`, download it from [here](https://github.com/stnava/ANTs/releases) and follow the instructions on the `ANTs` [wiki](https://github.com/stnava/ANTs/wiki/Compiling-ANTs-on-Linux-and-Mac-OS).
+1. If not listed, the pipeline does not require any additional dependency outside Clinica.
 
-#### Configuration
+<div markdown="1" class="third_party_table">
 
-We strongly recommend installing `ANTs >= 2.5.0` from which **no environment variable are needed**.
+|                      | ANTs | Convert3D | FreeSurfer | FSL | ITK | Matlab | MRtrix3 | PETPVC | SPM |
+|:--------------------:|:----:|:---------:|:----------:|:---:|:---:|:------:|:-------:|:------:|:---:|
+|    Anat > Volume     |      |           |            |     |     |   ✓    |         |        |  ✓  |
+|  Anat > FreeSurfer   |      |           |     ✓      |     |     |        |         |        |     |
+| DWI > Preprocessing  |  ✓   |     ✓     |            |  ✓  |     |        |    ✓    |        |     |
```

Comment:
I think we are missing some pipelines here... For example, I don't see t1-linear / flair-linear